Makefile

```diff
@@ -36,6 +36,10 @@ pre-production: test-cluster
 tps-sandbox: production-cluster
 	$(eval include global_config/tps-sandbox.sh)
 
+.PHONY: pentest
+pentest: production-cluster
```

Comment:
this will need to be
 pentest: platform-test-cluster
and that means you need to add the below to the Makefile
 platform-test-cluster:
       $(eval CLUSTER_RESOURCE_GROUP_NAME=s189t01-tsc-pt-rg)
       $(eval CLUSTER_NAME=s189t01-tsc-platform-test-aks)